src/translations/en_US/content/colinknock/fighter/messages.json

```diff
@@ -0,0 +1,49 @@
+{
+  "content.michaelmars.sinfulSailor": "Sinful Sailor",
```

Comment:
should this file be removed from this PR? it's for the fighter class, and the strings in it are for sinfulSailor.